suseconnect-ng.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Tue Sep 26 08:57:50 UTC 2023 - Miquel Sabate Sola <msabate@suse.com>
+
+- Update to version 1.4.0
+  * Added EULA display for addons (bsc#1170267)
+  * Fix zypper argument for auto-agreeing licenses (bsc#1214781)
+  * Enable building on SLE12 SP5
```

Comment:
```suggestion
  * Enable building on SLE12 SP5 (jsc#PED-3179)
```

This is important otherwise it will not get linked to the task and then discussions about SLE12 break loose!